ui/src/profiles/ProfileCoverImage.tsx

```diff
@@ -25,7 +26,8 @@ export default function ProfileCoverImage({
   children,
 }: CoverProps) {
   const contact = useContact(ship);
-  const { cover } = contact || emptyContact;
+  const calm = useCalm();
+  const cover = !calm.disableRemoteContent || contact || emptyContact;
```

Comment:
I think something went wrong here this should have given a type error. Missing destructuring, but also the logic needs to be different I think.